docs/plugins/bpsync.rst

```diff
@@ -0,0 +1,37 @@
+BPSync Plugin
+=============
+
+This plugin provides the ``bpsync`` command, which lets you fetch metadata
```

Comment:
Instead of duplicating the text from the MBSync plugin's docs, let's just briefly summarize what this plugin does and link to the MBSync page (by just saying "this plugin works similarly to…").